templates/adherent/committees.html.twig

```diff
@@ -4,74 +4,83 @@
-            <div class="committee-committee search__results__row l__row l__row--h-stretch l__row--v-center search__results__row--referent-committee">
-                <div class="search__results__info l__col l__col--v-stretch l__col--left">
-                    <div class="search__results__meta">
-                        <h2 class="text--normal text--medium b__nudge--bottom-small text--breakword">
-                            <a class="link--no-decor"
-                               href="{{ path("app_committee_show", { slug: membership.committee.slug }) }}">
-                                {{ membership.committee.name }}
-                            </a>
-                        </h2>
-                        <div>
-                            <span class="text--small text--bold">{{ membership.committee.getCityName }}</span>
-                            {{ 'committee.members_count'|transchoice(membership.committee.membersCount) }}
-                        </div>
+            <div class="committee-committee adherent__activity comittee space--30">
+                <div class="committee__details b__nudge--bottom-50">
+                    <h2 class="text--normal text--medium b__nudge--bottom-small text--breakword">
+                        <a class="link--no-decor"
+                           href="{{ path("app_committee_show", { slug: membership.committee.slug }) }}">
+                            {{ membership.committee.name }}
+                        </a>
+                    </h2>
+                    <div>
+                        <span class="font-roboto text--bold">{{ membership.committee.getCityName }}</span>
+                        <span class="font-roboto">{{ 'committee.members_count'|transchoice(membership.committee.membersCount) }}</span>
                     </div>
-
-                    <div class="search__results__cta">
+                </div>
+                <div class="committee__actions">
+                    <div class="left">
+                        <a href="{{ path("app_committee_show", { slug: membership.committee.slug }) }}"
+                           class="btn btn--blue b__nudge--right-small" target="_blank" rel="noopener noreferrer">
+                            Voir
+                        </a>
                         <button role="button"
-                                class="btn btn--blue committee-action-button"
+                                class="btn btn--ghosting--blue committee-action-button"
                                 id="committee-toggle-membership-{{ loop.index0 }}"
                                 data-action="quitter"
                                 data-slug="{{ membership.committee.slug }}"
                                 data-csrf-token="{{ csrf_token('committee.unfollow') }}">
                             <img src="{{ asset('images/loader-sm.gif') }}" class="btn-loader visually-hidden" />
-                            Quitter ce comité
```

Comment:
Seen with Guillaume, the wording was to be changed to "Quitter ce comité", to keep consistency with the "show committee" action iirc.